string.go

```diff
@@ -28,3 +31,18 @@ func Substring[T ~string](str T, offset int, length uint) T {
 func RuneLength(str string) int {
 	return utf8.RuneCountInString(str)
 }
+
+// Chop returns str with the last character removed.
+// If str ends with \r\n, both characters are removed.
```

Comment:
Bit it could end with \n and it won't work?

What happens if the last character is space or tab, or multiple \n or \r\n ?

I'm unsure this behavior would be expected.

Also if str is "a\r\nb", what do you expect? "a", "a\r\n" ?

I mean right now you code trim \r\n or the last character.

I'm unsure I would use a method with such behavior. What is the need that you are trying to solve?